test/SkipCallsTestHook.t.sol

```diff
@@ -55,7 +55,9 @@ contract SkipCallsTest is Test, Deployers, GasSnapshot {
 
     function test_beforeInitialize_skipIfCalledByHook() public {
         SkipCallsTestHook skipCallsTestHook = SkipCallsTestHook(
-            address(uint160(type(uint160).max & clearAllHookPermisssionsMask | Hooks.BEFORE_INITIALIZE_FLAG))
+            address(
+                uint160(type(uint160).max & clearAllHookPermisssionsMask | Hooks.BEFORE_INITIALIZE_FLAG) + 100000000000
```

Comment:
If the mask actually cleared the hook flags the addition would result in an overflow, remove.